Assets/Scripts/Data Model/Ecology/FishingDistributionSet.cs

```diff
@@ -3,45 +3,47 @@
 
 using System.Collections.Generic;
 
-public class FishingDistributionSet
+namespace MSP2050.Scripts
 {
-	private Dictionary<string, Dictionary<int, float>> fishingValues;
-
-	public FishingDistributionSet(FishingDistributionDelta initialValues)
+	public class FishingDistributionSet
 	{
-		fishingValues = new Dictionary<string, Dictionary<int, float>>(initialValues.FleetCount);
-		foreach (KeyValuePair<string, Dictionary<int, float>> fleetValues in initialValues.GetValuesByFleet())
+		private Dictionary<string, Dictionary<int, float>> fishingValues;
+
+		public FishingDistributionSet(FishingDistributionDelta initialValues)
 		{
-			Dictionary<int, float> distributionValues = new Dictionary<int, float>(fleetValues.Value);
-			fishingValues.Add(fleetValues.Key, distributionValues);
+			fishingValues = new Dictionary<string, Dictionary<int, float>>(initialValues.FleetCount);
+			foreach (KeyValuePair<string, Dictionary<int, float>> fleetValues in initialValues.GetValuesByFleet())
+			{
+				Dictionary<int, float> distributionValues = new Dictionary<int, float>(fleetValues.Value);
+				fishingValues.Add(fleetValues.Key, distributionValues);
+			}
 		}
-	}
 
-	public void ApplyValues(FishingDistributionDelta deltaSet)
-	{
-		foreach (KeyValuePair<string, Dictionary<int, float>> values in deltaSet.GetValuesByFleet())
+		public void ApplyValues(FishingDistributionDelta deltaSet)
 		{
-			Dictionary<int, float> target = fishingValues[values.Key];
-			foreach (KeyValuePair<int, float> value in values.Value)
+			foreach (KeyValuePair<string, Dictionary<int, float>> values in deltaSet.GetValuesByFleet())
 			{
-				target[value.Key] = value.Value;
+				Dictionary<int, float> target = fishingValues[values.Key];
+				foreach (KeyValuePair<int, float> value in values.Value)
+				{
+					target[value.Key] = value.Value;
+				}
 			}
 		}
-	}
 
-	public IEnumerable<KeyValuePair<string, Dictionary<int, float>>> GetValues()
-	{
-		return fishingValues;
-	}
+		public IEnumerable<KeyValuePair<string, Dictionary<int, float>>> GetValues()
+		{
+			return fishingValues;
+		}
 
-	public Dictionary<int, float> FindValuesForFleet(string fleetName)
-	{
-		Dictionary<int, float> result;
-		fishingValues.TryGetValue(fleetName, out result);
-		return result;
-	}
+		public Dictionary<int, float> FindValuesForFleet(string fleetName)
+		{
+			Dictionary<int, float> result;
+			fishingValues.TryGetValue(fleetName, out result);
+			return result;
+		}
 
-	/*public void NormalizeValues()
+		/*public void NormalizeValues()
```

Comment:
Commented code, as always see if it makes sense to leave it there for the future, otherwise remove it! :)